hq/app/logging/Cloudwatch.scala

```diff
@@ -24,15 +25,32 @@ object Cloudwatch extends Logging {
     val iamCredentialsWarning = Value("iam/credentials/warning")
     val iamKeysTotal = Value("iam/keys/total")
     val sgTotal = Value("securitygroup/total")
+    val gcpTotal = Value("gcp/total")
+    val gcpCritical = Value("gcp/critical")
+    val gcpHigh = Value("gcp/high")
+  }
+
+  def logMetricsForGCPFindings(allGcpFindings: Seq[GcpFinding]): Unit = {
+    val gcpProjectToFinding = allGcpFindings.groupBy(_.project)
+
+    gcpProjectToFinding.toSeq.foreach {
+      case (project: String, findings: Seq[GcpFinding]) =>
```

Comment:
@philmcmahon Is there a better way to iterate over a list of tuples and get variable names for each side of it?

Reply:
I think this is a good way of doing it!